<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>painter</class>
 <widget class="QWidget" name="painter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>painter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="painterLayout">
     <item>
      <widget class="QScrollBar" name="painterScrollBar">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="sliderPosition">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="lineChangedGroup">
       <property name="title">
        <string>曲线参数操作</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QComboBox" name="selectLIneComboBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="selectLIneButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>显示</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="showAllButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>显示全部</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hideAllButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>隐藏全部</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="viewGroup">
         <property name="title">
          <string>区间显示</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="uperTimeEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>——</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="downTimeEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="showRegionTimeButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>显示区间</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="lineGroup">
       <property name="title">
        <string>曲线显示操作</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="pauseShowButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>开始显示</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="autoAbjustButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>自动调整</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="clearLineButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>清除表格</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="viewAllButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>显示全时刻</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="longestTimeGroup">
       <property name="title">
        <string>最长显示时间</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>一页最长显示</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="longestTimeEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>S</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
